# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1544810404 28800
#      Fri Dec 14 10:00:04 2018 -0800
# Node ID 73512571d85af59900b43f063b9e42bed2e4a668
# Parent  00924e6e513e83d4cf4377c40f888ea41694e605
__line__.m: remove debugging commands from cset 23b56a61c918

* __line__.m: Remove keyboard command.

diff --git a/scripts/plot/draw/private/__line__.m b/scripts/plot/draw/private/__line__.m
--- a/scripts/plot/draw/private/__line__.m
+++ b/scripts/plot/draw/private/__line__.m
@@ -139,17 +139,16 @@ function h = __line__ (hp, varargin)
       ##        the style if the options in other_args specify a color
       ##        or linestyle.  The plot will be made correctly, but the next
       ##        call to line may not use the correct value.
       [linestyle, marker] = __next_line_style__ ();
       if (nr == 1)
         ## Marker for a single point is always '.' (bug #38825).
         marker = '.';
       endif
-      keyboard;
       color = __next_line_color__ ();
 
       handles(i) = __go_line__ (hp, data_args{:},
                                 "color", color, "linestyle", linestyle,
                                 "marker", marker, other_args{:});
     endfor
   endif
 
